src/globus_sdk/login_flows/login_flow_manager.py

```diff
@@ -30,15 +24,15 @@ class LoginFlowManager(metaclass=abc.ABCMeta):
 
     def __init__(
         self,
-        login_client: AuthLoginClient,
+        login_client: globus_sdk.AuthLoginClient,
         *,
         request_refresh_tokens: bool = False,
         native_prefill_named_grant: str | None = None,
     ) -> None:
-        if not isinstance(login_client, NativeAppAuthClient) and not isinstance(
-            login_client, ConfidentialAppAuthClient
-        ):
-            raise GlobusSDKUsageError(
+        if not isinstance(
+            login_client, globus_sdk.NativeAppAuthClient
+        ) and not isinstance(login_client, globus_sdk.ConfidentialAppAuthClient):
```

Comment:
`isinstance()` accepts a tuple of possible values, like `str.startswith()`:

```suggestion
        if not isinstance(login_client, (globus_sdk.NativeAppAuthClient, globus_sdk.ConfidentialAppAuthClient)):
```

Reply:
Yep; good call to tidy this up!

Reply:
I am 0% surprised that black reflowed this. (Since I plan to squash, I don't mind.)